dist/api_data.py

```diff
@@ -814,6 +814,15 @@ def __ge__(self, other):
                 directory must already exist. If the value is not an absolute path, the path
                 is relative to the database home (see @ref absolute_path for more information)'''),
         ]),
+    Config('rollback_to_stable', '', r'''
+        rollback tables to an earlier point in time, discarding all updates to checkpoint durable
+        tables that have durable times more recent than the current global stable timestamp''',
+        type='category', subconfig=[
+            Config('threads', 4, r'''
+                maximum number of threads WiredTiger will start to help RTS. Each
+                RTS worker thread uses a session from the configured session_max''',
+                min=0, max=10),
```

Comment:
Is there any way to combine this with the `WT_CONNECTION.rollback_to_stable` API definition?

At a minimum, I suggest adding a comment and a #define in `rollback_to_stable.h` to define the min and max values that both share. Look in `api_data.py` here for lines that have `Must match...` for several examples. This will be important for session allocation. See below.

Also, there is a computation when allocating the session array that includes additional calculations for internal sessions needed. So I think the 2nd sentence is not correct and slightly misleads the user. This value should be part of that internal calculation. See below.

Reply:
Thanks Sue, since RTS runs exclusively, I think we don't need to modify `conn_session_size` to reserve extra slots like we do for others (e.g., LSM threads, eviction threads). What do you think?